internal/database/mariadb/support_group.go

```diff
@@ -370,3 +370,52 @@ func (s *SqlDatabase) RemoveUserFromSupportGroup(supportGroupId int64, userId in
 
 	return err
 }
+
+func (s *SqlDatabase) GetSupportGroupNames(filter *entity.SupportGroupFilter) ([]string, error) {
+	l := logrus.WithFields(logrus.Fields{
+		"filter": filter,
+		"event":  "database.GetSupportGroupNames",
+	})
+
+	baseQuery := `
+    SELECT SG.supportgroup_name FROM SupportGroup SG
```

Comment:
This will return multiple times the same support group name no? 

I think we are missing a group by?